README.md

```diff
@@ -23,6 +24,24 @@ Read more about build-info and build integration in Artifactory [here](https://w
 
 ## Using build-info-go as a CLI
 
+### Clone and Build the CLI
+
+Clone the build-info-go project by executing the following command:
+```
+git clone https://github.com/jfrog/build-info-go
+```
+Build the project by navigating to the project folder and executing the following commands.
+
+On Unix based systems run:
+````
+./buildscripts/build.sh
+````
+On Windows run:
+````
+.\buildscripts\build.bat
+````
+Once completed, you will find the Build-Info CLI executable at the project directory.
+
 ### Generating Build-Info
 
 The Build-Info CLI lets you generate basic build-info for your project easily and quickly.
```

Comment:
The Build-Info CLI lets you generate basic build-info for your project easily and quickly.
-->
The Build-Info CLI allows generating build-info for your project easily and quickly.